compiler/one-cmds/one-profile

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+''''export SCRIPT_PATH="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)" # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
+''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+''''exit 255                                                                            # '''
+
+# Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import glob
+import itertools
+import ntpath
+import os
+import subprocess
+import sys
+import tempfile
+
+import utils as _utils
+
+
+def _get_backends_list():
+    """
+    [one hierarchy]
+    one
+    ├── backends
+    ├── bin
+    ├── doc
+    ├── include
+    ├── lib
+    └── test
+
+    The list where `one-profile` finds its backends
+    - `bin` folder where `one-profile` exists
+    - `backends` folder
+
+    NOTE If there are backends of the same name in different places,
+     the closer to the top in the list, the higher the priority.
```

Comment:
`closer to the top in the list` : what would be the sort order?

Reply:
At first, I wondered how same name of backends can exist. I didn't know `bin` folder is also searched. Then, `NOTE` seeming aims for giving priority between `bin` and `backends/*`. If so, I think we can make the sentence clear.

Also, I guess backends are not searched recursively for both `bin` and `backends`. Am I right?

Reply:
> Also, I guess backends are not searched recursively for both bin and backends. Am I right?

Yes, right. `one-profile` searches the *-profile file in `./bin` first, then in `backends/*`. There is no recursive search case.

I can't understand the expression the same name of backends.
I think it means that if there are multiple backends in backends directory, their priority is in alphabetical order.

I will update it.